web-app/src/app/_services/lap.service.ts

```diff
@@ -29,7 +29,7 @@ export class LapService {
   updateLapData(data: ILapDataInterface): void {
     const newData = new LapData();
     newData.lapNumber = data.lapnumber;
-    newData.lapTime = data.timestamp;
+    newData.lapTime = '22:22:00';
```

Comment:
Was this meant to be related to the data somehow?

Reply:
lol, I totally forgot about that we already have a `secondsdifference` field.